src/_posts/2025-08-06-the-ukpa-needs-you.md

```diff
@@ -0,0 +1,39 @@
+
+## What is the UKPA?
+
+The UK Python Association is a registered charity (registered in England and Wales, charity no. [1173471](http://apps.charitycommission.gov.uk/Showcharity/RegisterOfCharities/CharityWithoutPartB.aspx?RegisteredCharityNumber=1173471&SubsidiaryNumber=0)) dedicated to advancing education in the Python programming language in the UK and beyond. Established to support and grow the Python community, the UKPA serves as the driving force behind the annual PyCon UK conference and other initiatives that promote Python education and collaboration.
+
+The UKPA exists to promote Python education across all levels, from beginners taking their first steps to experienced developers sharing advanced techniques. We foster community growth by creating opportunities for Python enthusiasts to connect, learn, and collaborate, while supporting diversity and inclusion in the tech community to ensure Python is accessible to everyone. 
+
+## How can you get involved?
+
+The UKPA exists for the Python community - that’s you! And we’d love you to get involved.  The UK Python community is only as strong as its members - and right now, we need your help to grow even stronger.
+
+### Become a Member
+
+When you purchase your PyCon UK conference ticket, you have the option to become a UKPA member. As a member, you can attend our Annual General Meeting (AGM), which takes place during the PyCon UK conference, and vote for trustees who will guide the organisation's future.
+
+There's no additional cost associated, just opt in during ticket purchase.
+
+### Become a Trustee
+
+Are you ready to take your involvement to the next level? We currently have trustee vacancies and are looking for passionate community members to help guide the UKPA's future. As a trustee, you can help shape the strategic direction of the organisation, work with fellow trustees to ensure PyCon UK continues to grow and improve, contribute to decisions about grants, community initiatives, and educational programmes, and gain valuable experience in charity governance and event organisation.
+
+We particularly welcome applications from underrepresented groups in tech, as diverse perspectives strengthen our decision-making and better serve our community.
+
+#### How to Apply
+
+If you’d like to put your name forward as a trustee, send an email to trustees@uk.python.org by **Sunday 21st August 2025**. Please include a brief overview of your background and involvement in the Python community, what skills and perspective you'd bring to the board, and your vision for the future of Python education and community building in the UK.
+
+Your application will be acknowledged by email, and voting for new trustees will take place at the UKPA’s AGM, during the PyCon UK conference. 
+
+The UK Python Association has been supporting the Python community in the UK and beyond for over 8 years. Join us to help shape our processes, goals and aspirations for the years to come. 
```

Comment:
If we wanted to make this reusable for future years, we could put a UKPA trustee applications closing date in the site config, and then say 'since <date>' rather than 'for over 8 years'. I'm not sure if that's worth doing?

Reply:
Fair nok!